### ISSUE TITLE
Weekly Guild Quest Bug

### ISSUE THREAD

**Issue by Orbys** [ValveSoftware/Dota2-Gameplay#10184]
### Description

Frequently occurring bug that causes the weekly guild quest not to trigger even if you have 3 or more guild mates in your party. 

This has been occurring for 3 years now.

**Comment by Aluerie** [comment]
duplicate #75

**Comment by Orbys** [comment]
Yeah I am aware... but its been an issue for 3  years... so if they are not going to fix it might as well keep bringing it up.
________________________________
